src/fides/api/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -591,6 +591,7 @@ def get_ctl_datasets(
     only_unlinked_datasets: bool = False,
 ) -> List[Dataset]:
     """
+    Deprecated. Use `GET /datasets` instead.
```

Comment:
You can add a `deprecated` flag to the router info, here's an [example](https://github.com/ethyca/fides/blob/main/src/fides/api/api/v1/endpoints/privacy_request_endpoints.py#L689):

```
@router.get(
    PRIVACY_REQUESTS,
    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_READ])],
    response_model=Page[
        Union[
            PrivacyRequestVerboseResponse,
            PrivacyRequestResponse,
        ]
    ],
    deprecated=True,
)
def get_request_status(
```

Then it will be nice and obvious in the Swagger docs
![image](https://github.com/user-attachments/assets/02ef80fe-3b92-4b3b-b113-e9783de5e302)

Reply:
ah cool! didn't know this 😎